Lib/test/support/__init__.py

```diff
@@ -108,6 +108,18 @@
     "run_with_tz", "PGO", "missing_compiler_executable", "fd_count",
     ]
 
+
+# True if Python is running on Microsoft Windows.
+MS_WINDOWS = (sys.platform == 'win32')
```

Comment:
Wouldn't be better to use `is_windows` and `is_macos` for uniformity with `is_jython` and `is_android`?

Or rename the latter two?

Reply:
I replied here: https://github.com/python/cpython/pull/7800#issuecomment-398688993

If we do something, I would prefer to rename is_android and is_jython to ANDROID and JYTHON to be more consistent with PEP 8 and existing code (rather than adding support.is_windows and support.is_macos).